Can_Node_Handler.cpp

```diff
@@ -75,6 +75,10 @@ void Can_Node_Handler::handleMessage(Frame& message) {
     writeThrottleMessages(0);
     return;
   }
+  
+  int16_t raw_RPM = Store().readMotorRpm(0);
```

Comment:
Try to use the actual Motor enum type, rather than an integer - it's hard for someone reading this code to understand what 0 means.